docs/OTAnalytics/usage-ui.md

```diff
@@ -0,0 +1,434 @@
+
+The [tracker in OTVision](../../OTVision/usage/track) links consecutive detections, or bounding boxes, to form a **Track** or **Road User**.
+A track is therefore composed of a series of detections, each with its own classification.
+Ideally, all detections within a track share the same classification.
+However, in practice, a track may include detections with varying classifications (e.g., truck and truck_with_trailer).
+This typically occurs when only a portion of the road user is visible in one frame, while in subsequent frames more of the road user or the entire road user appears within the frame.
+
+OTAnalytics assigns a single **Track Classification** to each track, as a road user can have only one classification in the real world (e.g., a vehicle is either a car or a bus).
+
+To analyze the tracks, OTAnalytics provides sections and flows. **Sections** are detectors.
+Each time a track's trajectory intersects a section, an **Event** is generated.
+Each event contains detailed information, including the track's identifier, its position in the frame, and the timestamp of the intersection.
+
+A track's trajectory is created by selecting a single representative point from each bounding box associated with the track.
+This point is called the **Track Point**.
+The trajectory is formed by connecting the track points collected across successive frames.
+
+!!! info inline end "Track Point examples"
+    === "0.1, 0.1"
+        ![Offset x=0.1, y=0.1](usage-ui/offset-x-0-1-y-0-1.png){ align=left }
+
+    === "0.8, 0.3"
+        ![Offset x=0.8, y=0.3](usage-ui/offset-x-0-8-y-0-3.png){ align=left }
+
+The position of the track point within the bounding box can be configured using the **Offset** attribute of sections.
+This offset is defined separately for the x- and y-axes, with values ranging from `0` to `1`.
+These values determine the relative position of the track point within the bounding box, providing flexibility in track point placement.
+
+Since the offset can be individually configured for each section, tracks intersecting different sections can be finely tuned.
+This allows precise customization of the trajectory data based on the requirements of each section.
+
+**Flows** define traffic flows to be analyzed.
+As explained in [Flows](#flows), a flow is defined by two sections: a starting section and an ending section.
+Tracks that intersect both sections of a flow can be assigned to that flow.
+To achieve this, the events belonging to a track are sorted chronologically based on their time of occurrence.
+The pair of events with the longest time interval between them is used to assign the track to the flow corresponding to the sections of both events.
+The assignment of a track to a flow is called **Track Assignment**.
+
+## User Interface
+
+The user interface is divided into four main areas: the configuration bar, workspace, visualization layers, and filter area.
```

Comment:
You can switch between ‘Canvas’ and ‘Files’ in the UI. ‘Canvas’ encompasses many things: the actual canvas and the “Track statistics” (which you refer to as “Workspace”), the visualisation filters and the visualisation layers. ‘Canvas’ is therefore somewhat misleading. However, I would leave it as it is now and rebuild the UI from scratch in nicegui later on